schemas/aerleon-config.schema.json

```diff
@@ -1,52 +1,51 @@
 {
-  "$id": "urn:aerleon:schemas:aerleon-config:1.0.0",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "aerleon.yml",
-  "description": "Used to configure Aerleon's aclgen, aclcheck, and cgrep command line tools.",
+  "description": "configure Aerleon's aclgen, aclcheck, and pol2yaml command line tools",
```

Comment:
Some descriptions start with upper case and others lowercase. Same for punctuation. Lets fix that?

Reply:
Better yet, some of these top-level descriptions can be removed.